src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2191,6 +2129,32 @@ private void CheckForLatestRender(object obj)
             this.VisualizationManager.CheckIfLatestAndUpdate((long)obj);
         }
 
+        private void StartPeriodicTimer(object parameter)
+        {
+            if (Model == null)
+                return;
+
+            model.StartPeriodicEvaluation(HomeSpace.RunSettings.RunPeriod);
+        }
+
+        private bool CanStartPeriodicTimer(object parameter)
+        {
+            return true;
+        }
+
+        private void StopPeriodicTimer(object parameter)
+        {
+            if (Model == null)
+                return;
+
+            model.StopPeriodicEvaluation();
+        }
+
+        private bool CanStopPeriodicTimer(object parameter)
+        {
+            return true;
+        }
+
```

Comment:
This will all need to be moved to `HomeWorkspaceModel` when we have multiple home workspaces.